src/stories/textarea/fd-textarea.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { moduleMetadata } from '@storybook/angular';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { FormControlDirective, FormModule } from 'libs/core/src/lib/form/public_api';
+
+export default {
+    title: 'Fd text-area',
+    component: FormControlDirective,
+    moduleMetadata: moduleMetadata,
+    decorators: [
+        withKnobs,
+        withA11y,
+        moduleMetadata({
+            imports: [FormModule, ReactiveFormsModule],
+            declarations: []
+        })
+    ]
+};
+
+export const TextArea = () => ({
+    template:
+        `
+        <div fd-form-item>
+        <label fd-form-label for="input-1">Default Text Area</label>
+        <textarea fd-form-control id="id" [attr.aria-label]="ariaLabel" [placeholder]="placeholderVar" [disabled]="disabledVar" [compact]="compactVar" [state]="stateVar"></textarea>
+        </div>
+  `,
+    props: {
+        stateVar: select('State', {
```

Comment:
avoid using `Var` suffix, it is a good practice to give variable more meaningfull name (but without suffix like Var, List, Array)
ie. 
- state
- buttonOptions
- discountPerProductIncludingClientDiscount

Reply:
fixed